invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/deposit.html

```diff
@@ -16,19 +16,19 @@
 
 {%- block page_body %}
   {%- if record %}
-  <input id="deposits-record" type="hidden" name="deposits-record" value='{{record | tojson }}'></input>
+  <input id="deposits-record" type="hidden" name="deposits-record" value='{{record | tojson }}'/>
```

Comment:
isn't it JSX syntax?
```suggestion
  <input id="deposits-record" type="hidden" name="deposits-record" value='{{record | tojson }}'>
```
I think input in pure HTML does not have the end `/`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

Reply:
Apparently it doesn't matter if it's there or not, I just prefer it for the readability... although I don't have very strong opinions about it, so LMK if it should be removed.